lib/network.rb

```diff
@@ -95,7 +99,7 @@ def register_runner(token)
     private
 
     def broadcast message
-      print "#{Time.now.to_s} | #{message}"
+      print "#{Time.now.to_s} | #{@runner} | #{message}"
```

Comment:
I am not a ruby expert, so could you please show me how to fix this issue and I will apply it to later ones